core/Macros.carp

```diff
@@ -258,6 +258,10 @@ the filename and module name are the same.")
 (defmacro ignore [form]
   (list 'let (array '_ form) (list)))
 
+(doc ignore* "Wraps all forms passed as an argument in an ignore call.")
```

Comment:
How about we link to the docs for `ignore` here?
```suggestion
(doc ignore* "Wraps all forms passed as an argument in a call to [`ignore`](#ignore).")
```

Reply:
thanks!